OpenOversight/migrations/versions/6199adca9784_.py

```diff
@@ -0,0 +1,285 @@
+"""empty message
+
+Revision ID: 6199adca9784
+Revises: 0acbb0f0b1ef
+Create Date: 2018-10-28 18:46:59.078113
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '6199adca9784'
+down_revision = '0acbb0f0b1ef'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('departments',
```

Comment:
I feel like the upgrade/downgrade steps here are way bigger than they should be; could you try it again after making sure the DB was created properly? I _think_ it should just be something like
```suggestion
    op.alter_column('assignments', 'star_no', nullable=True)
```

Reply:
Sure, I will look into it.  Also, I got a new computer and I will be creating the DB from scratch, anyway.  I'm part of the docker on linux crew now!